website/content/_static/wireit.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 200 55" fill="none" stroke="#000" stroke-width="2" stroke-linecap="round">
```

Comment:
Consider instead copying from the version at the root of the repo?

Reply:
In this case I did the pure white version for dark theme. The outlined version doesn't look very good to me when small, because the detail is too fine.